src/ol_infrastructure/infrastructure/aws/policies/__main__.py

```diff
@@ -13,38 +13,39 @@
         }
     ]
 }
-
 describe_instance_policy = iam.Policy(
     'describe-ec2-instances-policy',
     name='describe-ec2-instances-policy',
     path='/ol-operations/describe-ec2-instances-policy/',
-    policy=json.dumps(describe_instance_policy_document),
+    policy=lint_iam_policy(describe_instance_policy_document, stringify=True),
```

Comment:
Why are we running the linter here and not when the policy is originally defined similar to what you've done below with the cloudwatch policy?